estate/models/estate_property.py

```diff
@@ -0,0 +1,38 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
```

Comment:
Odoo guidelines would say to first import external libraries, and then odoo ones.